### ISSUE TITLE
[feature] Configure static set of headers for proxy endpoints

### ISSUE THREAD

**Issue by oleg-smith** [centrifugal/centrifugo#684]
I want to be able to configure basic authorization header for Centrifugo proxy calls to backend.

Currently, I can only configure which headers to proxy from the original request.

**Describe the solution you'd like**.

As proxies are configured independently, the proposal is to add server configuration like this:

```
"proxy_{endpoint}_http_header_values" : {
   "Authorization": "Basic abc123",
   ...
}
```

Centrifugo adds these headers to proxied http requests.

Header values from `proxy_http_headers` will have a priority over the new config in case of a conflict.

...

**Comment by FZambia** [comment]
@oleg-smith hello,

In #687 I added such an option. One thing is that at this point we can't configure different headers for proxy on endpoint basis on configuration top-level. So on top-level we can introduce `proxy_static_http_headers` option. It will affect all the proxy requests. But! We have [granular proxy mode](https://centrifugal.dev/docs/server/proxy#granular-proxy-mode) - in that case headers may be configured per individual proxy endpoint.

Please take a look - do you have any questions?

**Comment by oleg-smith** [comment]
@FZambia looks great, thanks!

**Comment by oleg-smith** [comment]
Checking, if this will work for RPC proxies?

**Comment by FZambia** [comment]
Yes, 

> proxy_static_http_headers option ... will affect all the proxy requests.

**Comment by FZambia** [comment]
Released as part of [Centrifugo v5.0.2](https://github.com/centrifugal/centrifugo/releases/tag/v5.0.2)
